core/standalone/README.md

```diff
@@ -0,0 +1,153 @@
+<!--
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+# OpenWhisk Standalone Server
+
+OpenWhisk standalone server is meant to run a simple OpenWhisk server for local development and test purposes. It can be
+executed as a normal java application from command line.
+
+```bash
+java -jar openwhisk-standalone.jar
+```
+
+This should start the OpenWhisk server on port 3233 by default. Once the server is started then [configure the cli][1]
+and then try out the [samples][2].
+
+This server by default uses a memory based store and does not depend on any other external service like Kafka and CouchDB.
+It only needs Docker and Java to for running.
+
+Few key points related to it
+
+* Uses in memory store. Once the server is stopped all changes would be lost
+* Bootstraps the `guest` and `whisk.system` with default keys
+* Supports running on LacOS, Linux and Windows setup
+* Can be customized to use any other storage like CouchDB
+
+
+### Build
+
+To build this standalone server run
+
+```bash
+$ ./gradlew :core:standalone:build
+```
+
+This would create the runnable jar in `core/standalone/build/libs/` directory
```

Comment:
maybe `gradle ...:run` to then start the built jar?

Reply:
Good point. Now you run the jar directly via `$ ./gradlew :core:standalone:run`